docroot/modules/custom/govcms_base/govcms_base.module

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Contains module code for govCMS Base.
+ */
+
+use Drupal\views\Plugin\views\query\QueryPluginBase;
+use Drupal\views\ViewExecutable;
+
+/**
+ * Implements hook_views_query_alter().
+ */
+function govcms_base_views_query_alter(ViewExecutable $view, QueryPluginBase $query) {
+  // Filter the views by the search term in the query.
+  if ($view->id() == "knowledge_base") {
+    if (empty($_GET['title'])) {
+      return;
+    }
+    // Look for the title field.
+    $found = FALSE;
+    foreach ($query->where as &$where) {
+      if (empty($where['conditions'])) {
+        continue;
+      }
+      foreach ($where['conditions'] as &$condition) {
+        if ($condition['field'] == "node_field_data.title") {
+          $condition['value'] = "%" . $_GET['title'] . "%";
```

Comment:
I think we should use the request stack instead of accessing the super global.

``` php
$_GET['title'];
```
To
``` php
\Drupal::requestStack-> getCurrentRequest()->get('title');
```

I don't think we need to escape this string because the query alter is using the PDO but do you think we should just to be sure?